docs/exampleLink.mdx

```diff
@@ -0,0 +1,7 @@
+export const ExampleLink = ({component}) =>(
+	<>
+		<h2> Beispiele ansehen </h2>
+  	<a href ={`https://public-ui.github.io/v2/sample-react/#/${component}`}>Hier klicken, um das Biespiel der {component}-Komponente anzusehen </a>
```

Comment:
```suggestion
  	<a href={`https://public-ui.github.io/v2/sample-react/#/${component}`}>Beispiel der Komponente ansehen</a>
```

* Typo in `Beispiel`.
* Den Namen der Komponente würde ich hier weglassen, da wir nur den "technischen", nicht den gut lesebaren Namen der Komponente, haben.
* "Hier klicken" sollte im allgemeinen in Links vermieden werden. Durch den restlichen Text ist es vermutlich in Ordnung, persönlich würde ich es weglassen. Opinionated, entscheide du gerne.